doc/guide/packages.xml

```diff
@@ -118,6 +118,21 @@ $ cockpit-bridge --packages
         <term>version</term>
         <listitem><para>An informational version number for the package.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>preload</term>
+        <listitem><para>A list of identifiers (see below) of the
+        componented that should be preloded.  Normally, the files of a
+        component are loaded when the user navigates to it for the
+        first time.  The files of a preloaded component are loaded
+        immediately after the user logs in, and the initialization
+        code of the component is invoked.
```

Comment:
Can the page tell already if it's being initialized for preload or for the "full" page view? Or do we add that later?

Reply:
A page can check `cockpit.visibility`.  It probably needs to do that anyway since on preload, it will have size 0x0 and it needs to call `$('body').show()` again when it becomes visible...

Reply:
Nice! Could the playground example do this, as an illustration? And show a little paragraph or other element, so that the test case can check that this is not visible after preloading, but becomes visible after switching to the page?

Reply:
Done!